tests/wasm/runtime/index.ts

```diff
@@ -132,7 +109,10 @@ function imports(abi: () => Abi, host: Host): WebAssembly.Imports {
       'log.log': (level: number, message: Pointer) => {
         host.log.log(level, readStr(message))
       },
-      'store.get': () => 0,
+      'store.get': () => {
+        // TODO(nlordell): Implement storage reading
```

Comment:
nit: in larger projects I find it nice to create and link the task to make sure the TODO is being tracked.